docs/raster_datasets.md

```diff
@@ -0,0 +1,68 @@
+# Raster datasets
+
+OQT operates on raster datasets stored on disk.
+
+
+## Location (Data directory)
+
+The raster files are located in the `data` directory at the root of git repository.
+
+The location can be changed by setting the environment variable `$OQT_DATA_DIR`.
+
+For testing purposes the `data` directory at the root of the git repository contains raster files *clipped* to the extent of Heidelberg.
+The `data` directory is listed in `.gitignore`. This way the original raster files in full size can be stored in this directory for development.
```

Comment:
I guess this would be also important for deployment. maybe for deployment even more than for development

Reply:
Yes. That we will discuss how to setup properly.